docs/index.md

```diff
@@ -35,4 +35,5 @@ provider "konnect-beta" {
 - `konnect_access_token` (String, Sensitive)
 - `personal_access_token` (String, Sensitive)
 - `server_url` (String) Server URL (defaults to https://global.api.konghq.com)
+- `service_access_token` (String, Sensitive)
```

Comment:
We should remove this via overlay as it's an internal token

Reply:
Sent a PR - https://github.com/Kong/platform-api/pull/1682